x-pack/plugins/maps/public/classes/layers/tiled_vector_layer/tiled_vector_layer.tsx

```diff
@@ -64,13 +63,16 @@ export class TiledVectorLayer extends VectorLayer {
     );
     const prevDataRequest = this.getSourceDataRequest();
 
-    const canSkip = await canSkipSourceUpdate({
```

Comment:
we can remove this check. There is no app-state that influences stale-ness. All the relevant state for 3rd party mvt services, is embedded in the source-params.